hack/verify-generated-docs.sh

```diff
@@ -10,18 +10,15 @@ source "${OS_ROOT}/hack/common.sh"
 cd "${OS_ROOT}"
 
 GENERATED_DOCS_ROOT="${OS_ROOT}/docs/generated"
-_tmp="${OS_ROOT}/_tmp"
-TMP_GENERATED_DOCS_ROOT="${_tmp}/generated"
+TMP_GENERATED_DOCS_ROOT="$${OS_ROOT}/_tmp/generated"
```

Comment:
> The diff is run on generated, so this is functionally equivalent, right

You need to deconflict paths under _tmp.  Consider parallel execution cases or cases where multiple update commands are run and we need to debug.  Having conflicting paths makes that harder.